### PR TITLE
8279520: SPNEGO has not passed channel binding info into the underlying mechanism

### PR DESCRIPTION
**Pull request by wangweij**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8279520](https://bugs.openjdk.java.net/browse/JDK-8279520): SPNEGO has not passed channel binding info into the underlying mechanism

### Reviewers
 * [Sean Mullan](https://openjdk.java.net/census#mullan) (@seanjmullan - **Reviewer**)
 * [Valerie Peng](https://openjdk.java.net/census#valeriep) (@valeriepeng - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6969/head:pull/6969` \
`$ git checkout pull/6969`

Update a local copy of the PR: \
`$ git checkout pull/6969` \
`$ git pull https://git.openjdk.java.net/jdk pull/6969/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6969`

View PR using the GUI difftool: \
`$ git pr show -t 6969`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6969.diff">https://git.openjdk.java.net/jdk/pull/6969.diff</a>

</details>

### CODE REVIEW

**Comment by valeriepeng** [comment]
I will take a look.

File: src/java.security.jgss/share/classes/sun/security/jgss/spnego/SpNegoContext.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2021, Oracle and/or its affiliates. All rights reserved.
```
Comment (seanjmullan) [inline src/java.security.jgss/share/classes/sun/security/jgss/spnego/SpNegoContext.java:2]: 2022 now :)

**Review by seanjmullan — APPROVED** [review]

**Review by valeriepeng — APPROVED** [review]
Looks good. Thanks.

**Pull request closed without merging**